netmanager/src/config/constants.js

```diff
@@ -65,6 +65,8 @@ const prodConfig = {
   "http://34.78.78.202:30002/api/v1/devices/delete/components?comp=",
   UPDATE_COMPONENT_URI: 
   "http://34.78.78.202:30002/api/v1/devices/update/components?device=",
+  GET_online_offline_maintenance_status: 
+  "http://34.78.78.202:30002/api/v1/monitor/devices/online_offline",
```

Comment:
Kindly update the production node port  for device-monitoring microservice to `30006`

Reply:
Thanks, I have made the change